backend/login/views.py

```diff
@@ -1,10 +1,13 @@
-from django.shortcuts import render
 from rest_framework.decorators import api_view
 from rest_framework.response import Response
-from .use_cases import match_email_to_id
+from .use_case import LoginUseCase 
```

Comment:
Ohh thank you, this is a good point! I wasn't sure about how to call that because, unlike in Java, I don't think we can declare an abstract type but then use the concrete implementation in Python, but I will look into it!